lib/assets.js

```diff
@@ -1,10 +1,10 @@
 'use strict';
 
-var merge = require('lodash').merge;
-var gulpif = require('gulp-if');
-var imagemin = require('gulp-imagemin');
+import {merge} from 'lodash';
+import gulpif from 'gulp-if';
+import imagemin from 'gulp-imagemin';
 
-var defaults = {
+const defaults = {
```

Comment:
> I don't think this is incorrect so much as it implies that these are the final, unchangeable settings that this task will use when in fact they get munged in with args. So, it can be argued that the defaults object is constant, yes, technically, but I don't know that this is in the spirit of the const keyword?

To clarify...I wouldn't necessarily _argue_ for using `const` here, I was just trying to say that if it were used for anything in this script, the `defaults` object seems the most acceptable. I'm not sure how acceptable this use case is in general. I usually only use `const` for primitives, for example:

```js
const SOME_INTERVAL = 100;
const SOME_URL = 'http://example.com';
```